mocha.js

```diff
@@ -776,7 +776,7 @@ module.exports = function(suite) {
   });
```

Comment:
Wow — good catch! I’ve checked `git log` and it looks like `mocha.js` is rebuild quite frequently just before the release. That’s good, I can drop those changes than and we’ll get a simpler change! 🤓